src/components/Login.js

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from "react";
+const Login = (props) => {
+  const {
+    email,
+    setEmail,
+    confirmPassword,
+    setConfirmPassword,
+    confirmPasswordError,
+    password,
+    setPassword,
+    forgotPassword,
+    setForgotPassword,
+    handleLogin,
+    handlesignUp,
+    hasAccount,
+    setHasAccount,
+    emailError,
+    passwordError,
+    resetPassword,
+    loading,
+    setLoading,
+    user,
+    setUser
+  } = props;
+
+  return (
+    <section className="login">
+      <div className="loginContainer">
+        <h1 className="head" align="center">
+          Welcome{" "}
```

Comment:
what's the purpose of the `{"  "}`?

Reply:
That would not effect the Functioning of the page .The presence of {" "} is for future update on the Welcome message if needed.